app/assets/stylesheets/application.css.scss

```diff
@@ -40,11 +40,13 @@ body {
   font-size: 1.5em;
 }
 
-// .btn {
-//   background: #9ECAE1;
-//   color: white;
-//   font-weight: bold;
-// }
+a:hover {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.